### DISCUSSION TITLE
Label orientation with custom edges

### DISCUSSION THREAD

**Discussion by RococoCoding** [xyflow/xyflow#1175] [Q&A] [answered]
How do I get labels in custom edges to show horizontally following the x axis instead of following the path? 
![Screen Shot 2021-05-05 at 9 38 55 PM](https://user-images.githubusercontent.com/63833718/117230107-4fefcc00-adea-11eb-9d2f-d4784880800d.png)

**Comment by moklick** [comment, marked as answer]
that custom edge is an example of what you can do with custom edges. Maybe this example is a better one for your use case https://github.com/wbkd/react-flow/blob/main/example/src/Edges/CustomEdge2.tsx

**Reply by RococoCoding** [reply]
Ooooh Thank you so much! I didn't realize you had more examples in the repo.
